### ISSUE TITLE
undefined method `exists?' for File:Class

### ISSUE THREAD

**Issue by LFDMCreations** [sickill/racksh#16]
Hi, 
Using racksh in a Sinatra + Sequel project. In my gemfile:
```
group :test, :development do
  gem 'racksh', '~> 1.0'
end
```
however running `racksh` from root runs into a problem:
```
❯ racksh
/Users/thiebo/.rvm/gems/ruby-3.2.1/gems/racksh-1.0.0/lib/racksh/init.rb:28:in `init': undefined method `exists?' for File:Class (NoMethodError)

      eval(File.read(rcfile)) if File.exists?(rcfile)
                                     ^^^^^^^^
Did you mean?  exist?
	from /Users/thiebo/.rvm/gems/ruby-3.2.1/gems/racksh-1.0.0/bin/racksh:19:in `<top (required)>'
	from /Users/thiebo/.rvm/gems/ruby-3.2.1/bin/racksh:25:in `load'
	from /Users/thiebo/.rvm/gems/ruby-3.2.1/bin/racksh:25:in `<main>'
	from /Users/thiebo/.rvm/gems/ruby-3.2.1/bin/ruby_executable_hooks:22:in `eval'
	from /Users/thiebo/.rvm/gems/ruby-3.2.1/bin/ruby_executable_hooks:22:in `<main>'
```

**Comment by dentarg** [comment]
FYI, I merged this https://github.com/sickill/racksh/pull/15 over at https://github.com/spinels/racksh and published `spinels-racksh` (https://rubygems.org/gems/spinels-racksh)

**Comment by sickill** [comment]
Fixed in https://github.com/sickill/racksh/pull/15

**Comment by sickill** [comment]
Fix published in 1.0.1: https://rubygems.org/gems/racksh/versions/1.0.1
